src/main/resources/db/migration/V1_21__operational_capacity.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE operational_capacity
+(
+    id serial not null
+        constraint operational_capacity_pk primary key,
+    capacity int not null default 0,
+    prison_id varchar(3) not null unique,
+    date_time timestamp not null,
+    approved_by varchar(255) not null
+);
```

Comment:
I suggest we seed all the prisons in this data set with zero's to start off with

Reply:
Do you think we should prepare some initial sql to fill data in as a part of current job ? 
if yes. Can you provide the full list of prisons as I don't know them